packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -1459,4 +1461,22 @@ describe('Dropdown', () => {
       expect(stopPropagation).toBeCalledTimes(1)
     })
   })
+
+  describe('label', () => {
+    it('is rendered with the text passed as prop', () => {
+      const labelText = 'Add People'
+      const wrapper = mountWithProvider(<Dropdown items={items} label={labelText} />)
+      const label = getLabelWrapper(wrapper)
+
+      expect(label.exists()).toBe(true)
```

Comment:
hmm why is `findIntrinsicElement` needed? maybe I'm missing something. Can you try
```typescript
const label = wrapper.find(`.${Dropdown.slotClassNames.label}`)
expect(label.exists()).toBe(true)
```

Reply:
No, it does not. That selector finds the component as well.